loopchain/rest_server/rest_server_rs.py

```diff
@@ -84,7 +84,7 @@ def set_resource(self):
     def get_peer_list(self, channel):
         return self.__stub_to_rs_service.call(
             "GetPeerList",
-            loopchain_pb2.CommonRequest(request="", group_id=conf.ALL_GROUP_ID, channel=channel))
+            loopchain_pb2.CommonRequest(request="", group_id="all_group_id", channel=channel))
```

Comment:
This group ID is not deleted.

Reply:
`group_id` is partially removed here. Group ids related to grpc proto will be updated at the next pull request.